app/models/permission.rb

```diff
@@ -99,6 +99,8 @@ class Permission < ApplicationRecord
 
     metric.read
 
+    package.read
```

Comment:
Good time to put together migrations for adding a new permission to all roles. Right now, we only have a Rake task for adding new permissions to admins. We'll need to add tasks for the other roles as well.

Reply:
A simple way to do this would be to insert a `package.read` permission for every role and token that has a `artifact.read` permission.